spot_driver/spot_driver/robot_command_util.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2023-2024 Boston Dynamics AI Institute LLC. See LICENSE file for more info.
+
+"""
+Utility class with methods to manipulate robot commands.
+"""
+from typing import Any, List
+
+from bosdyn.api import robot_command_pb2
+from bosdyn.util import duration_to_seconds
+
+
+def is_batch_required(command: robot_command_pb2.RobotCommand, batch_size: int) -> bool:
```

Comment:
Nit: This both checks if batching is required and if it's allowed.  Maybe `should_batch`?

Reply:
Renamed.